docs/aqua-book/changelog.md

```diff
@@ -10,16 +10,16 @@ Aqua compiler's versioning scheme is the following: `0.BREAKING.ENHANCING.RELEAS
 ### [0.14.7](https://github.com/fluencelabs/aqua/releases/tag/aqua-v0.14.7) - May 14, 2024
 
 
-* **compiler:** Allow declaring imported functions. See [Functions](./language/expressions/functions.md) and [Headers](./language/header/header.md). ([LNG-357](https://linear.app/fluence/issue/LNG-357)) ([#1127](https://github.com/fluencelabs/aqua/issues/1127))
-* **compiler:** Allow declaring imported services and abilities. See [Abilities](./language/abilities.md), [Services](./language/services.md) and [Headers](./language/header/header.md). ([LNG-360](https://linear.app/fluence/issue/LNG-360)) ([#1135](https://github.com/fluencelabs/aqua/issues/1135))
-* FIX: **lsp:** Correct go-to-definition in VSCode for used types. See [`use`](./language/header/header.md) and [VSCE](./getting-started/installation/installation.md). ([LNG-345](https://linear.app/fluence/issue/LNG-345)) ([#1128](https://github.com/fluencelabs/aqua/issues/1128))
+* **compiler:** Allow declaring imported functions. See [Functions](./language/expressions/functions.md) and [Headers](./language//expressions/header.md). ([LNG-357](https://linear.app/fluence/issue/LNG-357)) ([#1127](https://github.com/fluencelabs/aqua/issues/1127))
+* **compiler:** Allow declaring imported services and abilities. See [Abilities](./language/abilities.md), [Services](./language/services.md) and [Headers](./language/expressions/header.md). ([LNG-360](https://linear.app/fluence/issue/LNG-360)) ([#1135](https://github.com/fluencelabs/aqua/issues/1135))
+* FIX: **lsp:** Correct go-to-definition in VSCode for used types. See [`use`](./language/expressions/header.md) and [VSCE](./getting-started/installation/installation.md). ([LNG-345](https://linear.app/fluence/issue/LNG-345)) ([#1128](https://github.com/fluencelabs/aqua/issues/1128))
```

Comment:
linear link doesn't work for me if i'm not logged into linear.

Reply:
yeah, previously thought about it. Seems like the only way to completely solve this is to just remove this links, but decided to leave them so at least we can use them. Also I am not completely sure if this list is complete and if it's gonna remain there for long anyway
In this pr just wanted to fix broken links for this, that's all